crates/runtime/src/model/params/bedrock.rs

```diff
@@ -0,0 +1,33 @@
+/*
+Copyright 2024-2025 The Spice.ai OSS Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+use super::{COMMON_MODEL_PARAMETERS_WITH_DEPRECATED, PARAM_WITH_DEPRE_LEN, concat_arrays};
+use crate::parameters::ParameterSpec;
+
+pub(crate) const BEDROCK_PARAMETERS: [ParameterSpec; 3] = [
+    ParameterSpec::runtime("aws_secret_access_key")
```

Comment:
To keep parity with our other AWS components (i.e. https://spiceai.org/docs/components/data-connectors/dynamodb) these should be `component` parameters, not runtime parameters.

Reply:
we would need to change it for embedding models too